src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestV4ServiceImpl.java

```diff
@@ -224,8 +231,102 @@ protected void updateRequestStatusInConcur(
                 (resultsDTO.getProcessingResults() == ConcurEventNotificationVersion2ProcessingResults.validAccounts);
         LOG.info("updateRequestStatusInConcur, Will notify Concur that Request " + requestUuid
                 + " had an overall validation of " + isValid);
+        if (!shouldUpdateStatusInConcur()) {
+            LOG.info("updateRequestStatusInConcur, Concur workflow actions are currently disabled "
+                    + "in this KFS environment");
+            return;
+        }
+        
+        String requestId = resultsDTO.getReportNumber();
+        String workflowAction = isValid ? ConcurWorkflowActions.APPROVE : ConcurWorkflowActions.SEND_BACK;
+        String logMessageDetail = buildLogMessageDetailForRequestWorkflowAction(
+                workflowAction, requestId, requestUuid);
+        
+        ConcurWebRequest<ConcurRequestV4ReportDTO> webRequest = buildWebRequestForTravelRequestWorkflowAction(
+                workflowAction, requestUuid, resultsDTO);
+        
+        ConcurRequestV4ReportDTO updatedTravelRequest = concurEventNotificationV2WebserviceService.callConcurEndpoint(
```

Comment:
I think it makes total sense to use this service.  I wonder if we should change the name of the service since it is no longer used just for event notification.  That may be something we should do in a follow up user story

Reply:
I thought the service was still being used for event-notification-related purposes, both for reading reports/requests and performing workflow actions on reports/requests. Nevertheless, I agree that the service's name is confusing and that it makes sense to update it in the future. I'll create a follow-up user story for that.